app/models/manageiq/providers/embedded_automation_manager/configuration_script_source.rb

```diff
@@ -92,7 +92,7 @@ def checkout_git_repository(target_directory = nil)
     target_directory     ||= Pathname.new(Dir.mktmpdir)
 
     git_repository.update_repo
-    git_repository.checkout(scm_branch, target_directory)
+    git_repository.checkout(scm_branch, target_directory.to_s)
```

Comment:
Super-minor, but if we're going to to_s it anyway, then no need to create a Pathname object on line 92.  Alternatively, maybe git_repository.checkout can accept Pathname objects?

Reply:
You had commented on the original PR about the consistency between the yielded object and the returned object which I agreed with.  If we didn't want to to_s here or on 102 then we'd have to `return Pathname.new(target_directory)` and also `yield Pathname.new(target_directory)`.  LMK

Reply:
Oh I see - yeah, you yield the Pathname object...I missed that bit.  This is good then :+1: